libbpf-cargo/src/main.rs

```diff
@@ -101,7 +101,7 @@ enum Command {
 fn main() {
```

Comment:
What would the implications be to move main to return Result<()> ?

Reply:
Ah yeah good idea